backend/src/personnel/personnel.service.ts

```diff
@@ -625,7 +629,11 @@ export class PersonnelService {
 
     const lastDeployed = await this.getLastDeployedDate(id);
 
-    return person.toResponseObject(role, lastDeployed);
+    const personnel = person.toResponseObject(role, lastDeployed);
+    if (role === Role.LOGISTICS) {
```

Comment:
hrmm this shouldn't be included - I thought we had the class xformer omitting this...